admin-console/src/main/webapp/src/components/SystemBehaviorGroups/CreateEditBehaviorGroupModal.tsx

```diff
@@ -0,0 +1,85 @@
+import { ActionGroup, Button, Form, FormGroup,
+    FormSelect,
+    FormSelectOption,
+    HelperText, HelperTextItem, Modal, ModalVariant, TextInput } from '@patternfly/react-core';
+import React from 'react';
+
+import { BehaviorGroup } from '../../types/Notifications';
+
+interface CreateEditModalProps {
+    isEdit: boolean;
+    showModal: boolean;
+    isOpen: boolean;
+    initialSystemBehaviorGroup?: Partial<BehaviorGroup>;
+    isLoading: boolean;
+    onClose: () => void;
+    onSubmit: (systemBehaviorGroup: Partial<BehaviorGroup>) => void;
+}
+
+export const CreateEditBehaviorGroupModal: React.FunctionComponent<CreateEditModalProps> = (props) => {
+
+    const [ systemBehaviorGroup, setSystemBehaviorGroup ] = React.useState<Partial<BehaviorGroup>>(props.initialSystemBehaviorGroup ?? {});
+
+    const actionOption = [
+        <FormSelectOption key={ 0 } isPlaceholder label='Choose an action' />,
+        <FormSelectOption key={ systemBehaviorGroup?.id } label='Send an email to Users: All' value={ systemBehaviorGroup?.id } />,
+        <FormSelectOption key={ systemBehaviorGroup?.id } label='Send an email to Users: Admins' value={ systemBehaviorGroup?.id } />
+    ];
+
+    const handleChange = (value: string, event: React.FormEvent<HTMLInputElement> | React.FormEvent<HTMLTextAreaElement>
+        | React.FormEvent<HTMLSelectElement>) => {
+        const target = event.target as HTMLInputElement | HTMLSelectElement;
+        setSystemBehaviorGroup(prev => ({ ...prev, [target.name]: target.value }));
+    };
+
+    const onSubmitLocal = React.useCallback(() => {
+        props.onSubmit(systemBehaviorGroup);
+    }, [ systemBehaviorGroup, props ]);
+
+    React.useEffect(() => {
+        setSystemBehaviorGroup(props.initialSystemBehaviorGroup ?? {});
+    }, [ props.initialSystemBehaviorGroup ]);
+
+    return (
+        <React.Fragment>
+            <Modal
+                variant={ ModalVariant.medium }
+                title={ `${ props.isEdit ? 'Update' : 'Create'} your System Behavior Group` }
+                isOpen={ props.showModal }
+                onClose={ props.onClose }
+            ><Form isHorizontal>
+                    <FormGroup label='Group Name' fieldId='displayName' isRequired
+                        helperText={ props.isEdit ? <HelperText><HelperTextItem variant="warning" hasIcon>
+                                                    If this field is modified it may affect exisiting behavior.
```

Comment:
This is not really true for behavior groups. The problem with event types, applications and bundles is that we reffer them by name in the events - but behavior groups are always reffered by ID.

Reply:
do you mean having a displayName field does not really make sense? because BG are referred by ID. Should we auto-generate an ID here instead maybe?

Reply:
@josejulio thoughts on the above?

Reply:
Sorry for late reply - I meant that we don't need to display the warning here. 
It's fine being able to edit the `displayName`, lets just remove the warning.